src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/etherium/EtheriumType.java

```diff
@@ -5,14 +5,14 @@
 import net.minecraft.resources.ResourceLocation;
 
 /**
- * The etherium types.
+ * Enum for etherium type.
  */
 public enum EtheriumType implements ITranslatable {
-    LIGHT(0xff79a5ed), NEUTRAL(0xff2effab), DARK(0xff800000);
+    LIGHT(0xff7fa7ef), NEUTRAL(0xff3fffbf), DARK(0xff800000);
```

Comment:
Did you test the colors with TOP? are the bars visible?

Reply:
Yes they are.